src/dispatch/static/dispatch/src/case/TimelineTab.vue

```diff
@@ -77,6 +74,26 @@ export default {
       Util.exportCSV(items, this.name + "-timeline-export.csv")
       this.exportLoading = false
     },
+    formatToTimeZones(date) {
+      if (!date) return ""
+
+      let m = moment(date)
+      return `UTC: ${date}<br> PST: ${m
+        .tz("America/Los_Angeles")
+        .format("YYYY-MM-DD HH:mm:ss")}<br>EST: ${m
+          .tz("America/New_York")
+          .format("YYYY-MM-DD HH:mm:ss")}`
+    },
   },
 }
 </script>
+
+<style scoped>
+.wavy-underline {
+  text-decoration: underline;
+  text-decoration-style: wavy;
```

Comment:
I think this will look better, may need to up the darkness of the gray. I planned on changing this in the CasePage component as well. Let me know what you think.

```suggestion
  text-decoration-style: dotted;
```